Phase8/EFCore/SearchEngine/InvertedIndex.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using SearchEngine.Interfaces;
+
+namespace SearchEngine
+{
+    public class InvertedIndex : IInvertedIndex
```

Comment:
This is a long class with many responsibilities and can be broken down into 2 classes(Searcher, Tokenizer)